configure.ac

```diff
@@ -468,7 +468,8 @@ AC_ARG_ENABLE([ibnet],
 	[],
 	[enable_ibnet="check"])
 AM_CONDITIONAL([ENABLE_IBNET], [test "x$enable_ibnet" != xno])
-AS_IF([test "x$enable_ibnet" = xyes],[
+AS_IF([test "$enable_ibnet" = xyes],[
+	AC_MSG_NOTICE([Disable ibnet module NOT requested])
```

Comment:
I'm not sure that his notice makes sense to add.

Reply:
@morrone, no it's broken. I was playing with the github editor and accidentally broke it. the 'x' is missing, etc... @valleydlr has changes staged that he will test today and push tomorrow.